src/services/azureFunctionsService.ts

```diff
@@ -94,9 +95,17 @@ export class AzureFunctionsService implements mssql.IAzureFunctionsService {
 			folderPath: projectFile
 		});
 
+		// check for the new function file to be created
+		const functionFile = await newFilePromise;
+		const timeoutFunctionFile = await checkFunctionFile;
+		try {
+			await Promise.race([functionFile, timeoutFunctionFile]);
+		} catch (err) {
+			console.log(err);
```

Comment:
You shouldn't catch and just log here - we want this to fail if we hit the timeout since that implies the file wasn't detected and so trying to continue on will just result in an error (since we need to know the file path to update it...)